packages/expo-updates/ios/EXUpdates/Events/QueueUpdatesEventManager.swift

```diff
@@ -0,0 +1,71 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public class QueueUpdatesEventManager: UpdatesEventManager {
```

Comment:
```suggestion
internal class QueueUpdatesEventManager: UpdatesEventManager {
```

nit: would lean toward hide the implementation inside expo-updates and not exposing this publicly. if we can hide the `UpdatesEventManager` protocol as internal, that would be even better.

Reply:
Can't hide UpdatesEventManager since it's used in the controller interface which is used in the dev client interface (external interface)